api/proto/aws.proto

```diff
@@ -22,23 +22,14 @@ message AwsProvider {
     // may also need to set `AwsBakeryDefaults.awsSecretKey`.
     string secretAccessKey = 5;
 
-    // Halyard does not expose a command to edit this field, but defaults it to
-    // `{{name}}-keypair`.
-    // TODO(mneterval): Can we move to Clouddriver?
-    string defaultKeyPairTemplate = 6;
-
     // List of default regions.
-    repeated AwsRegion defaultRegions = 7;
-
-    // Defaults relevant to the AWS provider.
-    // TODO(mneterval): Can we move to Clouddriver?
-    AwsDefaults defaults = 8;
+    repeated AwsRegion defaultRegions = 6;
 
     // Configuration for AWS-specific features.
-    AwsFeatures features = 9;
```

Comment:
(I stared at this for about five years trying to figure out what looked off, then realized that I skipped `10` the first time around here.)

Reply:
Technically this is a breaking change from the proto perspective (re-numbering a field)...but given that we are only ever using this as an intermediary to serialize YAML (and nobody is using this yet) there should be no existing protos out there that this could break so I think this is fine.

Probably once we actually fully release this we'll want to stop re-numbering fields, but I think we should still do so where necessary now.

Reply:
Yes! And if we ever do need to remove fields in the future once we consider this a public API, we should use the [`reserved` keyword](https://developers.google.com/protocol-buffers/docs/proto#reserved) so new fields don't overwrite the number or name of removed fields.

Reply:
Oh, nice!  Adding the `reserved` keyword to the list of things I learned today...📝